batchflow/research/results.py

```diff
@@ -5,44 +5,18 @@
 import glob
 import json
 import dill
+import numpy as np
 import pandas as pd
 
+from .utils import to_list
+
 class Results:
```

Comment:
In my researches, if some class (for example, neural network block written as `nn.Module`) defined in the notebook itself, it raises errors on results loading. Is it possible to deal with it?